src/steem/asset.ts

```diff
@@ -36,6 +36,12 @@
 import * as assert from 'assert'
 import * as ByteBuffer from 'bytebuffer'
 
+export interface Nai {
```

Comment:
This is actually also an Asset but in the new "SMT" format, the steemd api's are a bit annoying now since they mix and match the old format and the new. Let's call this interface SMTAsset until we can migrate the Asset class completely

Reply:
Ok good call